src/services/redisSessionStorage.ts

```diff
@@ -301,6 +292,7 @@ export default class RedisSessionStorage extends RedisStorageUtils
   public async listUserSessions(
     user: User
   ): Promise<Either<Error, SessionsList>> {
+    await this.clearExpiredSetValues(user.fiscal_code);
```

Comment:
can you add a comment on why we need to update the set here ? (ie. session expire instead of logout)